dev-utils/src/commands/init_command.rs

```diff
@@ -0,0 +1,225 @@
+use super::genesis_command::GenesisCommand;
+
+use failure::Error;
+
+use super::genesis_dev_config::GenesisDevConfig;
+use read_input::prelude::*;
+use serde_json::json;
+use serde_json::Value as JsonValue;
+use std::fs;
+use std::fs::File;
+use std::io::prelude::*;
+use std::path::Path;
+use std::process::Command;
+use structopt::StructOpt;
+
+#[derive(Debug, StructOpt)]
+pub struct InitCommand {
+    data: String,
+    genesis_time: JsonValue,
```

Comment:
Any specific reason for this not to be `DateTime<Utc>`?

Reply:
originally, it's json based utility for read json, write json.
but i'm changing to use our core classes.

Reply:
but i'll change to core classes

Reply:
fixed